build.sbt

```diff
@@ -40,7 +43,6 @@ lazy val root = Project(libName, file("."))
     },
     generateExamplesManifest := {
       println("==========")
-      val _                      = generateExamples.value
```

Comment:
I see that this value was never used, but was this not the command to generate the examples? Or is it done aas a dependency of the `generateExamplesManifest` task?

Reply:
Nope, this just makes it re-generate the examples whenever generating the manifest, which was overwriting some manual changes I had been making.